oauth2client/contrib/appengine.py

```diff
@@ -89,11 +89,11 @@ def _safe_html(s):
     return cgi.escape(s, quote=1).replace("'", '&#39;')
 
 
-class InvalidClientSecretsError(Exception):
-    """The client_secrets.json file is malformed or missing required fields."""
+class Error(Exception):
+    """Base error for this module."""
 
 
-class InvalidXsrfTokenError(Exception):
+class InvalidXsrfTokenError(Error):
```

Comment:
Removing it.

On Thu, Jun 30, 2016, 11:02 AM Pat Ferate <notifications@github.com> wrote:

> In oauth2client/contrib/appengine.py
> <https://github.com/google/oauth2client/pull/533#discussion_r69180223>:
>
> >
> >
> > -class InvalidXsrfTokenError(Exception):
> > +class InvalidXsrfTokenError(Error):
>
> Are you talking about removing the InvalidXsrfTokenError? or keeping it's
> base class as Exception?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/google/oauth2client/pull/533/files/c21dd0d0fa9a0986cfe6001b399bd5be1b5bcc1c#r69180223>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAPUcxXh2CAXHXuLGhoUH4EQ8H004pT1ks5qRAPqgaJpZM4JBU6i>
> .
>